content/en/security/application_security/serverless/compatibility/_index.md

```diff
@@ -1,5 +1,7 @@
 ---
-title: Serverless Compatibility Requirements 
+title: Serverless Compatibility Requirements
+aliases:
+  - /security/application_security/enabling/compatibility/serverless
```

Comment:
This one appears to be broken?

Reply:
jk we're good

Reply:
works locally. I just added it after I created the PR so maybe it wasn't there yet when you tested the staged version.